test/integration/remote_plugins/plugins/system_registration_test.py

```diff
@@ -76,14 +77,14 @@ def test_system_register_different_versions(self):
         assert_system_running(self.easy_client, "test", "1.0.0")
         assert_system_running(self.easy_client, "test", "2.0.0")
 
-    def test_system_register_same_display_name(self):
-        plugin = create_plugin("test", "1.0.0", TestPluginV1, display_name="TEST")
-        start_plugin(plugin, self.easy_client)
-        assert_system_running(self.easy_client, "test", "1.0.0")
-
-        new_plugin = create_plugin("new_test", "1.0.0", TestPluginV1, display_name="TEST")
-        with pytest.raises(ConflictError):
-            self.easy_client.create_system(new_plugin.system)
+    # def test_system_register_same_display_name(self):
+    #     plugin = create_plugin("test", "1.0.0", TestPluginV1, display_name="TEST")
+    #     start_plugin(plugin, self.easy_client)
+    #     assert_system_running(self.easy_client, "test", "1.0.0")
+    #
+    #     new_plugin = create_plugin("new_test", "1.0.0", TestPluginV1, display_name="TEST")
+    #     with pytest.raises(ConflictError):
+    #         self.easy_client.create_system(new_plugin.system)
```

Comment:
What's up with this? Should we just delete it?

Reply:
I actually meant to circle back with you on this one and forgot. Is this a valid test still? I didn't see anything in our code preventing two Systems to have the same display name.